browser/js/common/factories/map.factory.js

```diff
@@ -1,45 +1,80 @@
-app.factory('MapFactory', function ($http) {
+app.factory('MapFactory', function($http) {
 
     var MapFactory = {};
-    MapFactory.findCenter = function(center){
+
+    MapFactory.findCenter = function(center) {
         center.replace(" ", "%20");
-        console.log(center);
         return $http.get("https://maps.googleapis.com/maps/api/geocode/json?address=new%20york%22&components=locality:" + center + "&key=AIzaSyDG1T1ZvOqHtAqgmkjjjHtXZz5HvwrW_w0")
-            .then(function(res){
-                var hoodInfo=res.data;
-                if (hoodInfo.results.length < 1) return {lat:40.705189, lng:-74.009209}
+            .then(function(res) {
+                var hoodInfo = res.data;
+                if (hoodInfo.results.length < 1) return {
+                    lat: 40.705189,
+                    lng: -74.009209
+                }
                 return hoodInfo.results[0].geometry.location;
             })
     }
+
     MapFactory.initialize_gmaps = function(center) {
 
         var styleArr = [{
             featureType: 'landscape',
-                stylers: [{saturation: -100}, {lightness: 60}]
+            stylers: [{
```

Comment:
any reason why this is all reformatted? I feel like it was a lot easier to read the other way